common/main/java/com/couchbase/lite/DataSource.java

```diff
@@ -101,7 +101,7 @@ String getColumnName() {
     @NonNull
     Map<String, Object> asJSON() {
         final Map<String, Object> json = new HashMap<>();
-        if (alias != null) { json.put("AS", alias); }
+        json.put("AS", getColumnName());
```

Comment:
While true, It could only do so if there is a db with no name, or a source that is not a Database.  I don't think either is possible, in the current code.